.travis.yml

```diff
@@ -3,17 +3,6 @@ language: scala
 jdk:
 - openjdk8
 
-matrix:
-  include:
-    - scala: 2.11.12
-      env: PLAY_VERSION=2.5
-    - scala: 2.11.12
-      env: PLAY_VERSION=2.6
-    - scala: 2.12.10
-      env: PLAY_VERSION=2.6
-     - scala: 2.12.10
-     env: PLAY_VERSION=2.7
-
 cache:
```

Comment:
the script should be changed to `sbt +test`

Reply:
Change made, pushed.